docs/content/ref/proxy.header.trust.xff.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "proxy.header.trust.xff"
+---
+
+`proxy.header.trust.xff` instructs fabio to trust the X-Forwarded-For header
```

Comment:
Maybe backtick `X-Fowarded-For` and `true` and `false`? I need to work on the CSS a bit more though.